lizmap/www/assets/js/dataviz/dataviz.js

```diff
@@ -295,7 +296,7 @@ var lizDataviz = function() {
         return pid;
     }
 
-    function buildPlot(id, conf){
+    function buildPlot(id, conf, pid = null){
```

Comment:
This change uses [Default parameters in function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters). The [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#browser_compatibility) seems OK with Lizmap 3.5 [browser compatibility](https://github.com/3liz/lizmap-web-client/wiki/Versions#web-browsers).